packages/atomic/src/components/common/generated-answer/rephrase-buttons.tsx

```diff
@@ -56,7 +56,10 @@ export const RephraseButtons: FunctionalComponent<RephraseButtonProps> = (
       <p part="rephrase-label" class="mb-2 text-neutral-dark shrink-0">
         {i18n.t('rephrase')}
       </p>
-      <div class="flex flex-wrap gap-2 ml-auto border border-solid rounded-md p-1">
+      <div
```

Comment:
I believe you should also document the part in `atomic-insight-generated-answer.tsx` and `atomic-generated-answer.tsx`

Reply:
aaagh good point I always forgot about that !!!